tests/mypy_test.py

```diff
@@ -213,57 +233,46 @@ def run_mypy(args: TestConfig, configurations: list[MypyDistConf], files: list[P
                 temp.write(f"{k} = {v}\n")
         temp.flush()
 
-        flags = get_mypy_flags(args, temp.name, testing_stdlib=testing_stdlib)
+        flags = [
+            "--python-version",
+            args.version,
+            "--show-traceback",
+            "--warn-incomplete-stub",
+            "--show-error-codes",
```

Comment:
Is this still needed now that its default in mypy?

Reply:
Good point, it's not!

```suggestion
```